framework/db/oci/Schema.php

```diff
@@ -621,7 +621,7 @@ public function insert($table, $columns)
                 } else {
                     $returnParams[$phName]['dataType'] = \PDO::PARAM_INT;
                 }
-                $returnParams[$phName]['size'] = isset($columnSchemas[$name]) && isset($columnSchemas[$name]->size) ? $columnSchemas[$name]->size : -1;
+                $returnParams[$phName]['size'] = isset($columnSchemas[$name], $columnSchemas[$name]->size) ? $columnSchemas[$name]->size : -1;
```

Comment:
Can we simplify `isset($columnSchemas[$name], $columnSchemas[$name]->size)` to `isset($columnSchemas[$name]->size)`? First check seems to be redundant.

Reply:
I noted the case for evaluation (and an inspection enhancement), but ATM can not say.

Reply:
Yes, we can. Could be made into inspection.